src/index.ts

```diff
@@ -343,11 +341,11 @@ export default class Zemu {
 
     while (inputSnapshotBufferHex.data.equals(currentSnapshotBufferHex.data)) {
       const currentTime = new Date()
-      const elapsed: any = currentTime.getTime() - start.getTime()
+      const elapsed = currentTime.getTime() - start.getTime()
       if (elapsed > timeout) {
         throw `Timeout waiting for screen to change (${timeout} ms)`
       }
-      Zemu.delay(500)
+      Zemu.delay()
```

Comment:
is it ok to decrease the delay from 500 to 250ms?

Reply:
Still testing on all platforms👌🏻